CHANGELOG.md

```diff
@@ -0,0 +1,30 @@
+# Change Log for MIGraphX
+
+Full documentation for MIGraphX is available at [MIGraphX Github Pages](https://rocmsoftwareplatform.github.io/AMDMIGraphX/doc/html/).
+
+## MIGraphX 2.5 for ROCm 5.5.0
+
+### Added
+- Y-Model feature to store tuning information with the optimized model
+- Accuracy checker tool based on ONNX Runtime
+- ONNX Operators parse_split, and Trilu 
+- Build support for ROCm MLIR
+- Added migraphx-driver flag to print optimizations in python (--python)
+
+
+### Optimizations
+- Improved performance of Transformer based models
+- Improved performance of the Pad, Concat, Gather, and Pointwise operators
+- Improved onnx/pb file loading speed
```

Comment:
- Added general `optimize` pass which runs several passes such as `simplify_reshapes`/`algebra` and DCE in loop.

Reply:
Done